freqtrade/freqtradebot.py

```diff
@@ -1088,21 +1222,28 @@ def _safe_exit_amount(self, pair: str, amount: float) -> float:
             raise DependencyException(
                 f"Not enough amount to exit trade. Trade-amount: {amount}, Wallet: {wallet_amount}")
 
-    def execute_trade_exit(self, trade: Trade, limit: float, sell_reason: SellCheckTuple) -> bool:
+    def execute_trade_exit(
+        self,
+        trade: Trade,
+        limit: float,
+        sell_reason: SellCheckTuple,  # TODO-lev update to exit_reason
+        side: str
```

Comment:
I don't think "side" should be a parameter here.
it's implicitly available via `trade.is_short` - so it provides no benefit, and most likely, every caller will use `'short' if trade.is_short else 'long'`.